### PR TITLE
Update 'influx export` and `influx stacks update` commands

### LINKED ISSUE #1154: Add '--stack-id' flag to 'influx export' command
PR: https://github.com/influxdata/influxdb/pull/18736

##### Relevant URLs
- https://v2.docs.influxdata.com/v2.0/reference/cli/influx/export/

### LINKED ISSUE #1157: Updates to the influx stacks update command
##### Relevant URLs
- PR: https://github.com/influxdata/influxdb/pull/18758

### PR DESCRIPTION
**Pull request by sanderson**
Closes #1154 
Closes #1157 

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/reference/cli/influx/stacks/update.md [hunk 1]
```diff
@@ -30,6 +31,16 @@ influx stacks update [flags]
 | `-u` | `--template-url`      | Template URLs to associate with a stack                    | list of strings |                       |
```
Comment (jsteenb2) [inline content/v2.0/reference/cli/influx/stacks/update.md:31]: I think teh `--export-file` is missing here 🤔

Comment (sanderson) [reply content/v2.0/reference/cli/influx/stacks/update.md:31]: Whoops, you're right.

File: content/v2.0/reference/cli/influx/export/_index.md [hunk 2]
```diff
@@ -57,4 +58,11 @@ influx export \
   --buckets=$BID1,$BID2,$BID3 \
   --labels=$LID1,$LID2,$LID3 \
   --dashboards=$DID1,$DID2,$DID3
+
+# Export all resources associated with a stack
+influx export --stack-id $STACK_ID
+
+# Export resources associated with a stack and resources
```
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/export/_index.md:65]: all resources in a bucket?

File: content/v2.0/reference/cli/influx/stacks/update.md [hunk 3]
```diff
@@ -30,6 +32,16 @@ influx stacks update [flags]
...
 | `-t` | `--token`             | Authentication token                                       | string          | `INFLUX_TOKEN`        |
 
+{{% warn %}}
+#### Export an updated template
+To prevent accidental changes, we **strongly recommend** exporting a new template
```
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/stacks/update.md:37]: To ensure all template resources are backed up, we...when you add resources...

File: content/v2.0/reference/cli/influx/stacks/update.md [hunk 4]
```diff
@@ -30,6 +32,16 @@ influx stacks update [flags]
...
+any time you add additional resources to a stack using the `--addResource` flag
+with the `influx stack update` command.
+The updated stack will differ from the previous template.
```
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/stacks/update.md:40]: Does this mean the updated stack includes a template with the newly added resources?

**Pull request merged**